boards/arduino-mkr1000/include/board.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2016 Inria
+ * Copyright (C) 2017 Inria
```

Comment:
Actually, I think convention is to use `2016-2017` when editing. Applies to many files of this PR